deeplake/integrations/pytorch/pytorch.py

```diff
@@ -99,7 +99,7 @@ def dataset_to_pytorch(
     for t in tensors:
         if dataset[t].is_sequence:
             raise NotImplementedError(
-                f"Deep Lake’s OSS pure-python dataloader is not compatible with tensor `{t}` with htype = sequence[…]. Please use the C++ dataloader via ds.dataloader(…), which can be installed using ‘pip install deeplake[enterprise]’."
+                f"Deep Lake’s OSS pure-python dataloader is not compatible with tensor `{t}` with htype = sequence[…]. Please use the C++ dataloader via ds.dataloader(…)"
```

Comment:
Instead of C++ dataloader, let's say 'high-performance dataloader'. That's consistent with language Levon is using in indra

Reply:
Yeah, I like that better than mentioning the language. Users don't care what we implement it in